basiclibrary/src/main/java/basiclibrary/Lab3.java

```diff
@@ -0,0 +1,59 @@
+package basiclibrary;
+
+import java.util.*;
+
+public class Lab3 {
+
+    public String tempFinder(int[][] inputArr) {
+        int min = 100;
+        int max = 0;
+        String ans = "";
+        HashSet<Integer> h = new HashSet();
+        for (int i = 0; i < inputArr.length; i++) {
+            for (int j = 0; j < inputArr[i].length; j++) {
+                h.add(inputArr[i][j]);
+                if (inputArr[i][j] > max)
+                    max = inputArr[i][j];
+                if (inputArr[i][j] < min)
+                    min = inputArr[i][j];
+            }
+        }
+        ans += ans.format("High: %s\nLow: %s\n", max, min);
+        for(int k = min; k < max; k++){
+            if(!h.contains(k)){
+                ans += ans.format("Never saw temperature: %s\n",k);
+            }
+        }
+        return ans;
+    }
+
+    public static String tally(List<String> arr){
+        HashMap<String, Integer> votes = new HashMap();
+        String[] plants = new String[]{"Bush", "Shrub", "Hedge"};
```

Comment:
This meets the requirements for the lab but means that this method can only tally votes for these three plantae